README.md

```diff
@@ -29,17 +29,19 @@ To get started working on Membership you will need to complete the following ste
 
 1. Go to the project root
 1. Run `./setup.sh` to install project-specific client-side dependencies
+1. Change the ownership of the 'gu' directory under 'etc' to current user.
+   `$ sudo -i chown -R {username} /etc/gu`
 1. Add the following to your hosts file in `/etc/hosts`:
 
 ```
 127.0.0.1   mem.thegulocal.com
 ```
 
-1. Run `./nginx/setup.sh`
+1. Run `membership-frontend$ ./nginx/setup.sh`
 1. Download our private keys from the `membership-private` S3 bucket. You will need an AWS account so ask another dev. If you have the AWS CLI set up you can run:
 
 ```
-aws s3 cp s3://membership-private/DEV/membership-keys.conf /etc/gu
+aws s3 cp s3://membership-private/DEV/membership-keys.conf /etc/gu --profile membership
```

Comment:
This command is dependent on the configuration of ~/.aws/credentials, so setting the `profile` argument is optional. Please indicate this in the readme.